src/main/java/core/basesyntax/impl/ParserImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Parser;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserImpl implements Parser {
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_PRODUCT_NAME = 1;
+    private static final int INDEX_OF_AMOUNT = 2;
```

Comment:
Let's try not to use `of` to make the names more shorter.
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int PRODUCT_NAME_INDEX = 1;
    private static final int AMOUNT_INDEX = 2;
```